test/game_teams_test.rb

```diff
@@ -10,7 +10,7 @@ def setup
     params = {
       game_id: "2012030221",
       team_id: "3",
-      HoA: "away",
+      hoa: "away",
```

Comment:
Atom was seeing "HoA" as a class due to first letter capitalized. Renamed - need to verify that column conversion to symbol still maps correctly.